doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -286,8 +286,8 @@ objects, and not to destroy them while they are still in use.
 
 Application-owned memory is immediately consumed by any Vulkan command it is
 passed into.
-The application can: alter or free this memory as soon as the commands that
-consume it have returned.
+The application must: not write to nor free this memory before the command
+that consumed it have returned.
```

Comment:
> I wish. You can though.

Yea ok that's actually fair. If we can find a way to say both things sanely, I'd be quite happy.